src/stores/utils.ts

```diff
@@ -0,0 +1,11 @@
+import type { State, StateCreator } from 'zustand';
+import create from 'zustand';
+import { devtools, subscribeWithSelector } from 'zustand/middleware';
+
+export const createStore = <T extends State>(name: string, storeFn: StateCreator<T>) =>
+  create(
+    devtools(subscribeWithSelector(storeFn), {
```

Comment:
Should the devtools stuff be wrapped with the `IS_DEV_BUILD` constant so that it's compiled away for prod?

Reply:
I thought that the middleware already did this, but it doesn't seem so. Thanks, I will change that!